helm/vro-app/templates/deployment.yaml

```diff
@@ -89,6 +89,7 @@ spec:
                   name: vro-secrets
                   key: VRO_SECRETS_SLACK
 
+            # These 2 should be moved to application-*.yml
```

Comment:
Reminder that these env variables don't need to be in a Helm config. See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings